docs/src/components/Layout/Page/Page.component.tsx

```diff
@@ -31,6 +31,18 @@ interface PageProps {
     enableFooter?: boolean;
 }
 
+// Defining this in one location so that it can be used both in the table styles for the pre tag,
+// and for the StyledInlineCode component fed to MDX's custom Components.
+const InlineCodeStyle = css`
+    display: inline;
+    background-color: rgba(27, 31, 35, 0.05);
+    font-family: 'SFMono-Regular', Consolas, Liberation Mono, Menlo, Courier,
+        monospace;
+    border-radius: 3px;
+    padding: 0.2em 0.4em;
+    font-size: 85%;
```

Comment:
I'm not a huge fan of mixing px, rem, and %-units.

Reply:
Yeah that's fair. This is a direct copy of the styles from Markdown's site, I was sloppy in updating it. Noted for my clean-up pass of the code and components.